src/promptflow/tests/executor/e2etests/test_batch_engine.py

```diff
@@ -118,27 +110,28 @@ def test_batch_storage(self, dev_connections):
         ],
     )
     def test_batch_run(self, flow_folder, inputs_mapping, dev_connections):
-        batch_result = submit_batch_run(flow_folder, inputs_mapping, connections=dev_connections)
+        batch_result, output_dir = submit_batch_run(
+            flow_folder, inputs_mapping, connections=dev_connections, return_output_dir=True
+        )
+
         assert isinstance(batch_result, BatchResult)
         nlines = get_batch_inputs_line(flow_folder)
-        msg = f"Bulk result only has {len(batch_result.line_results)}/{nlines} outputs"
-        assert len(batch_result.outputs) == nlines, msg
-        for i, output in enumerate(batch_result.outputs):
+        assert batch_result.total_lines == nlines
+        assert batch_result.completed_lines == nlines
```

Comment:
Add assert for `start_time` `end_time` `duration` here?

Reply:
Updated.